spec/integration/data_stream/_dev/test/system/config.spec.yml

```diff
@@ -34,6 +34,15 @@ spec:
           enum:
             - docker
           default: docker
+        type:
+          description: >
+            "Elastic Agent image to be used for testing. Setting default will be used the
+            same Elastic Agent image as the stack."
+          type: string
+          enum:
+            - default
+            - complete
+          default: "default"
```

Comment:
Complete would mean to use elastic-agent-complete image for docker.
Default the same image as the Elastic Agent from the stack.